src/atoms/StyledTextButton.tsx

```diff
@@ -19,12 +22,17 @@ export interface StyledTextButtonProps {
     placement?: GlobalMuiPlacement
   }
   IconRight?: JSX.Element // Try to give a space within the jsx element for better usage
+  sx?: SxProps<Theme>
 }
 
+// TODO: I need to consider how to place the theme because I did not want to introduce business related lambda OR state here.
+// TODO: But to bring the consistency, I need to consider the theme here.
+
```

Comment:
You should introduce something that has the theme too!
Maybe...
`src/ThemedTextButtonAtom.tsx`

Reply:
```
import { FC } from 'react'
import { getButtonColorLambda } from '@/lambdas/get-app-theme-color.lambda'
import { appThemeState } from '@/recoil/app-theme/app-theme.state'
import { useRecoilValue } from 'recoil'
import StyledTextButtonAtom, {
  StyledTextButtonProps,
} from '@/atoms/StyledTextButton'

const ThemedTextButtonAtom: FC<StyledTextButtonProps> = (props) => {
  const appTheme = useRecoilValue(appThemeState)

  return (
    <StyledTextButtonAtom
      sx={getButtonColorLambda(appTheme, props.isDisabled)}
      {...props}
    />
  )
}

export default ThemedTextButtonAtom

```

Reply:
Handled